src/components/Maps/MapPage.tsx

```diff
@@ -95,6 +100,17 @@ const MapPage = ({
     }
   });
 
+  useEffect(() => {
```

Comment:
I was running into issues with delete modal cancellation. Essentially, once I confirmed deletion, the info window was  still open. After lots of testing with @JasonYG, we decided to go with a `useEffect` solution and remove the usage of `handleCancel` for the maps confirmation modal.